CMakeLists.txt

```diff
@@ -43,4 +43,4 @@ include(${CMAKE_SOURCE_DIR}/cmake-modules/doxygen_common.cmake)
 # sub-projects
 add_subdirectory(sdk/core/azure-core)
 add_subdirectory(sdk/samples/http_client/curl) # will work only if BUILD_CURL_TRANSPORT=ON
-add_subdirectory(sdk/storage)
+#add_subdirectory(sdk/storage)
```

Comment:
uncomment this to re-enable storage cmake proj

Reply:
I think it's a very good practice to disable storage build when you introduce a breaking change in Azure Core.

Then why did you also change storage code?